css/docs.css

```diff
@@ -107,6 +108,24 @@
   color: var(--color-gray-dark);
 }
 
+.grid-item {
+  display: grid;
+  place-content: center;
+  padding: 1rem;
+  border-radius: 5px;
+  background-color: var(--color-default);
+  color: white;
+  font-size: 1.5rem;
```

Comment:
Font size too

Reply:
These styles are specific to the documentation site, so didn't use variables.